components/combobox/combobox.test.js

```diff
@@ -79,6 +82,33 @@ describe('DtCombobox Tests', function () {
       it('should render the input', function () { assert.isTrue(wrapper.find('input').exists()); });
     });
 
+    describe('When label is provided', function () {
+      beforeEach(async function () {
+        slots = { input: '<input v-bind="props.inputProps" />' };
+        _mountWrapper();
+        _setChildWrappers();
+      });
+
+      it('should provide proper label prop to input element', function () {
+        assert.equal(wrapper.find('input').attributes('label'), baseProps.label);
```

Comment:
You can just use `input` instead of `wrapper.find('input')` you already have it in `setChildWrappers`